Formula/simulate-pcr.rb

```diff
@@ -7,8 +7,11 @@ class SimulatePcr < Formula
 
   depends_on "bioperl"
   depends_on "blast"
+  depends_on "LWP::Simple" => :perl
```

Comment:
This was the previous now defunct way of adding Perl dependencies. It sadly no longer works. See for example `mlst` for how to add Perl dependencies to a formula.
https://github.com/brewsci/homebrew-bio/blob/f2cfb685f0be67b55601191b83fce12a71ed367a/Formula/mlst.rb#L26-L27